src/GoogleCloudStorageServiceProvider.php

```diff
@@ -78,6 +76,29 @@ public function boot()
         });
     }
 
+    /**
+     * Create a new StorageClient
+     *
+     * @param  mixed $config
+     * @return \Google\Cloud\Storage\StorageClient
+     */
+    private function createClient($config)
+    {
+        if (is_string($config['key_file'])) {
+            return new StorageClient([
+                'projectId' => $config['project_id'],
+                'keyFilePath' => $config['key_file'],
+            ]);
+        }
+        
+        return new StorageClient([
+                'projectId' => $config['project_id'],
+                'keyFile' => array_merge([
+                    "project_id" => $config['project_id']
+                ], $config['key_file'])
+            ]);
+        }
+    }
```

Comment:
this is a trailing bracket, composer will fail here upon package discovery.

Reply:
Thank you for reporting this issue.